protocol/directory.go

```diff
@@ -102,3 +121,66 @@ func (d *ConiksDirectory) Register(req *RegistrationRequest) (
 		return NewRegistrationProof(ap, d.LatestSTR(), nil, Success)
 	}
 }
+
+func (d *ConiksDirectory) KeyLookup(req *KeyLookupRequest) (
+	*DirectoryProof, ErrorCode) {
+	ap, err := d.pad.Lookup(req.Username)
+	if err != nil {
+		return nil, ErrorDirectory
```

Comment:
Ah ok. But currently, such an error is only handled in the registration case in `HandleOps`.